cmd/snapd-apparmor/main.go

```diff
@@ -68,19 +69,29 @@ import (
 // any loss of functionality. This is an unsupported configuration that cannot
 // be properly handled by this function.
 func isContainerWithInternalPolicy() bool {
+	var securityFSPath = filepath.Join(dirs.GlobalRootDir, "/sys/kernel/security")
+
 	if release.OnWSL {
 		// WSL-1 is an emulated Windows layer that has no support for AppArmor.
 		// WSL-2 is a virtualised environment with the Linux kernel as
-		// distributed by Microsoft.  In the future, if Microsoft enables
-		// AppArmor in their kernel configuration and adjusts their special
-		// init process, which launches individual distributions in
-		// quasi-containers, to initialize apparmor nesting, we might re-enable
-		// this and treat WSL-2 like a LXD/Incus container, with nested
-		// security.
+		// distributed by Microsoft.
+		//
+		// In the future, Microsoft might enable AppArmor in their kernel
```

Comment:
Nitpick: this very long sentence with many commas can changed to read easily.

e.g.
// In the future, Microsoft could enable AppArmor in the WSL kernel configuration
// and modify the special init process that launches distributions in quasi-containers.
// This change would initialize AppArmor nesting automatically, potentially eliminating
// the need for this WSL-specific logic.